cmd/test.go

```diff
@@ -310,8 +310,7 @@ func processWatcherUpdate(ctx context.Context, testParams testCommandParams, pat
 	})
 
 	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
+		fmt.Fprintln(testParams.output, err)
```

Comment:
Also a fix for case `2`, where we allow policy and data files to be only partially written to.

Reply:
Would it make sense to separate `compileAndSetupTests` and `runTests`. If the latter fails, we probably want to exit?

Reply:
And exit the watcher loop, you mean? I'm unsure. I'm almost thinking it make sense for the watcher to never exit unless manually forced to. As the user is probably continuously testing while actively developing; tests might compile and successfully run eventually 🤔 .